docker-compose.yml

```diff
@@ -51,8 +47,6 @@ services:
       - redis
       - citygram_services
     depends_on:
-      - db
-      - redis
       - citygram_migration
```

Comment:
why keep citygram_migration as a depends_on instead of a link?

Reply:
`link` setups a network link between two containers (allowing them to access each other). In the migration case, `citygram` doesn't need to talk to the migration container, it just wants it to have run.